orb.yml

```diff
@@ -8,62 +8,89 @@ examples:
   delta_analysis:
     description: >
       Run CodeScene Delta Analysis.
-      A context is used for specifying values for CODESCENE_HOST_URL, CODESCENE_USER, CODESCENE_PASSWORD and GITHUB_API_TOKEN.
+      A context is used for specifying values for CODESCENE_USER, CODESCENE_PASSWORD and GITHUB_API_TOKEN.
       The value for CODESCENE_DELTA_ANALYSIS_URL is project specific, and the value 
       it should be set to is available in the CodeScene project settings.
     usage:
       version: 2.1
       orbs:
-        codescene-ci-cd: empear/codescene-ci-cd
+        codescene-ci-cd: empear/codescene-ci-cd@x.y.z
       workflows:
         version: 2
         my-workflow:
           jobs:
             - codescene-ci-cd/run-delta-analysis:
-                # The context specifies values for:
-                #   CODESCENE_HOST_URL, CODESCENE_USER, CODESCENE_PASSWORD and GITHUB_API_TOKEN
                 context: codescene-ci-cd
-                codescene-user: ${CODESCENE_USER}
-                # CODESCENE_DELTA_ANALYSIS_URL is project specific. 
-                # The value it should be set to is available in the CodeScene project settings 
-                codescene-delta-analysis-url: ${CODESCENE_HOST_URL}${CODESCENE_DELTA_ANALYSIS_URL}
+
+  delta_analysis_with_custom_values:
+    description: >
+      Run CodeScene Delta Analysis.
+      Custom values for parameters are specified directly on the job.
+    usage:
+      version: 2.1
+      orbs:
+        codescene-ci-cd: empear/codescene-ci-cd@x.y.z
+      workflows:
+        version: 2
+        my-workflow:
+          jobs:
+            - codescene-ci-cd/run-delta-analysis:
+                codescene-delta-analysis-url: ${MY_DELTA_ANALYSIS_URL_ENV_VAR}
+                codescene-user: ${MY_CODESCENE_USER_ENV_VAR}
+                codescene-password: ${MY_CODESCENE_PASSWORD_ENV_VAR}
+                github-api-token: ${MY_GITHUB_API_TOKEN_ENV_VAR}
```

Comment:
I added an example of customizing parameters. 
I'm not sure I entirely understood your comment about parametrizing, but the way I've done it now these are just strings injected in the job during config processing (right?).